run-unit-test.sh

```diff
@@ -15,5 +15,6 @@ virtualenv env
 . env/bin/activate
 pip install -e .
 pip install nose mock coverage
+rm .coverage
```

Comment:
It's not cumulative, [see here](http://nedbatchelder.com/code/coverage/cmd.html#h_data_file).

> By default,each run of your program starts with an empty data set.